Omniture/README.md

```diff
@@ -19,4 +19,4 @@ Next, install the CocoaPods.
 pod install
 ```
 
-After installing the CocoaPods, add the Adobe SDK libraries to the `.xcworkspace` project file. Navigate into the mobile-services directory to find `AdobeMobileLibrary.a` for iOS and add it to the BasicOmniturePlayer project of BasicOmniturePlayer.xcworkspace. Navigate into the video-heartbeat-v2 directory to find `VideoHeartbeat.a` for iOS and add it to the BasicOmniturePlayer project of BasicOmniturePlayer.xcworkspace.
+After installing the CocoaPods, add the Adobe SDK libraries to the `.xcworkspace` project file. Navigate into the mobile-services directory to find `AdobeMobileLibrary.a` for iOS and add it to the BasicOmniturePlayer project of BasicOmniturePlayer.xcworkspace. Navigate into the video-heartbeat-v2 directory to find `VideoHeartbeat.a` for iOS and add it to the BasicOmniturePlayer project of BasicOmniturePlayer.xcworkspace. Finally, add `ADBMediaHeartbeat.h` to the project and in the project's Build Settings add the path to `ADBMediaHeartbeat.h` under `Header Search Paths`.
```

Comment:
nice update to the setup instructions

Reply:
Since the Omniture header search paths are known ahead of time (assuming the user follows the README), I'd prefer to have the paths already defined in the .xcodeproj the way the objc .xcodeproj does. Also, it's not necessary to add ADBMediaHeartbeat.h to the project because that header will never be edited by our users.

To spare you some digging, add these to the Header Search Paths of the target's settings:
```
"$(PROJECT_DIR)/mobile-services/sdks/iOS/AdobeMobileLibrary"
"$(PROJECT_DIR)/video-heartbeat-v2/sdks/iOS/libs/Headers"
```

Reply:
Done!